### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by DotNetDublin** [BlazingSaddlesSolutions/TimeKeeper#8]
I believe the main choices for logging within dot net applications is currently [NLog](https://nlog-project.org/) and [Serilog](https://serilog.net/) although I'd welcome any other suggestions.

Regardless of the logging platform implemented I would like the project to demonstrate when and what to log.

**Comment by harperjohn** [comment]
Yes - Logging is not "glamorous"  but it is extremely useful! (if not "essential")

**Comment by AnkurSheel** [comment]
Do we care about error reporting?
